fast_llm/engine/training/config.py

```diff
@@ -17,48 +25,85 @@
     from fast_llm.engine.training.trainer import Trainer
 
 
-def get_interval_config_class(desc: str, offset_desc: str | None = None):
-    # Intervals are a common pattern, so we standardize them with this helper.
-    @config_class()
-    class IntervalConfig(Config):
-        interval: int | None = Field(
-            default=None,
-            desc=f"The number of training iterations between each {desc}. Setting to None will disable.",
-            hint=FieldHint.feature,
-            valid=skip_valid_if_none(check_field(Assert.gt, 0)),
-        )
-        offset: int = Field(
-            default=0,
-            desc=f"Offset for the first {offset_desc or desc}.",
-            hint=FieldHint.feature,
-            valid=check_field(Assert.geq, 0),
+@config_class()
+class IntervalConfig(Config):
+    # Intervals are a common pattern, so we standardize them with this base class.
+    interval: int | None = Field(
+        default=None,
+        desc="The number of training iterations between each interval. Setting to None will disable.",
```

Comment:
maybe the wrong place to do this, but can we make it so that there is at least a warning that checkpoint saving is disabled? I don't think people will appreciate training for hours (or days) only to find out that they forgot to set a saving interval ;)

Reply:
That's an option, but would the warning actually be useful? A runtime warning is unlikely to be seen, and a validation one is only useful if the config is validated before launch.

Reply:
Hm, I see. Do you have a better idea?

Reply:
Not really....